File: iOS/GeoCoordinateConverterKit/GeoCoordinateConverterTests/TestData.m
//
//  LatLon2UTMTestData.m
//  GeoCoordinateConverter
//
//  Created by Volker Voecking on 30.06.14.
//  Copyright (c) 2014 VVSE. All rights reserved.
//

#import "TestData.h"

@implementation TestData

-(id) initWithName:(NSString*) name Latiude:(double)latitude Longitude:(double)longitude UTM:(NSString *)utm MGRS:(NSString *)mgrs {
    
    self = [super init];
    
    if ( self != nil ) {
    
        self.name = name;
        self.latitude = latitude;
        self.longitude = longitude;
        self.utm = utm;
        self.mgrs = mgrs;
    }
    
    return self;
}

@end

File: Android/Test/TestProject/src/TestData.java

public class TestData {

	public double latitude;
	public double longitude;
	public String utm;
	public String mgrs;
	public String name;

	public TestData( String name, double latitude, double longitude, String utm, String mgrs ) {
		
		this.name = name;
		this.latitude = latitude;
		this.longitude = longitude;
		this.utm = utm;
		this.mgrs = mgrs;
	}
}
